tests/py/dynamo/conversion/test_rsqrt_aten.py

```diff
@@ -25,6 +25,46 @@ def forward(self, input):
             inputs,
         )
 
+    @parameterized.expand(
+        [
+            (
+                "2d_dim_dtype_half",
+                (1, 1),
+                (2, 2),
+                (4, 4),
+                torch.half,
+                torch.half,
+            ),
+            (
+                "3d_dim_dtype_float",
+                (1, 1, 1),
+                (1, 2, 3),
+                (3, 3, 3),
+                torch.float,
+                torch.float,
+            ),
```

Comment:
Can you add a testcase with int32 as input and it returns float correct ?

Reply:
Thanks for pointing this out. Integer input has issue with both static/dynamic input.
below error from TRT when int input is passed to SQRT. I added cast to float
ERROR:torch_tensorrt [TensorRT Conversion Context]:3: [UNARY]-[aten_ops.rsqrt.default]-[rsqrt_default_sqrt]: operation SQRT not allowed on type Int32
https://github.com/pytorch/TensorRT/blob/main/py/torch_tensorrt/dynamo/conversion/impl/elementwise/ops.py#L124-L131